pkg/apis/machine/v1beta1/v1beta1_suite_test.go

```diff
@@ -51,13 +53,22 @@ var (
 )
 
 func TestMain(m *testing.M) {
+	validatingHooks := NewValidatingWebhookConfiguration()
+	mutatingWebhooks := NewMutatingWebhookConfiguration()
+
+	// Setting group version is required for testEnv to create unstructured objects, as the new structure sets it on empty strings
+	// Usual way to populate those values, is to create the resource in the cluster first, which we can't yet do.
+	validatingHooks.SetGroupVersionKind(admissionregistrationv1.SchemeGroupVersion.WithKind("ValidatingWebhookConfiguration"))
```

Comment:
why is this not set in `NewMutatingWebhookConfiguration`?

Reply:
It's more for tests than anything else. Why setting something we don't need otherwise in the main code?

Reply:
I think being explicit about the typeMeta is good regardless testing https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#resources